src/java/fr/paris/lutece/plugins/newsletter/business/portlet/NewsLetterSubscriptionPortlet.java

```diff
@@ -227,17 +225,17 @@ public String getXml( HttpServletRequest request )
         {
             if ( StringUtils.isNotEmpty( strMailError ) )
             {
-                XmlUtil.addElement( strXml, TAG_NEWSLETTER_EMAIL_ERROR, I18nService.getLocalizedString( PROPERTY_ERROR_INVALID_MAIL, getLocale( request ) ) );
+                XmlUtil.addElement( strXml, TAG_NEWSLETTER_EMAIL_ERROR, I18nService.getLocalizedString( PROPERTY_ERROR_INVALID_MAIL, AdminUserService.getLocale( request ) ) );

```

Comment:
AdminUserService.getLocale( request ) refers to BackOffice users.
The Portel.java class proposes  a  getLocale( request ) method , made for FO users.
So remove the static import, but keep that method call (and restrict in pom.xml the min version of CORE dependency)